src/pages/resultsView/ResultsViewPageHelpers.spec.ts

```diff
@@ -11,4 +12,46 @@ describe("ResultsViewPageHelpers", () => {
             assert.equal(getTabId("results/oncoprint/"), ResultsViewTab.ONCOPRINT);
         });
     });
+
+    describe('parseSamplesSpecifications', ()=>{
```

Comment:
there should be tests for other cases but need to get this out.  these test the issue that was broken